Bearded.UI/EventArgs/ModifierKeys.cs

```diff
@@ -19,6 +19,8 @@ private ModifierKeys(bool shift, bool control, bool alt, bool win)
             Win = win;
         }
 
+        public static ModifierKeys None { get; } = new ModifierKeys();
```

Comment:
Nit: 
```suggestion
        public static ModifierKeys None => new ModifierKeys();
```
For using less memory and fewer instructions.